frontend/src/components/recipe_card/recipe_card.js

```diff
@@ -4,24 +4,50 @@ import './recipe_card.css';
 
 
 const Recipe = (props) => {
+
+  const mealDifficulty = () => {
+    if (props.difficulty === "EASY") {
+      return (
+        <img className="icons" src="https://img.icons8.com/material-rounded/24/000000/low-connection.png" alt="Easy Difficulty Rating"></img>
+      )
+    }
+    if (props.difficulty === "MEDIUM") {
+      return (
+        <img className="icons" src="https://img.icons8.com/material-rounded/24/000000/medium-connection.png" alt="Medium Difficulty Rating"></img>
+      )
+    }
+    if (props.difficulty === "HARD") {
+      return (
+        <img className="icons" src="https://img.icons8.com/material-rounded/24/000000/bar-chart.png" alt="Hard Difficulty Rating"></img>
+      )
+    }
+  }
+
   return (
     <div className="card-deck d-inline-flex">
       <div className="card">
         <div className="image-holder">
-          <img className="card-img-top" src={props.name} alt={props.alternate} />
+          <img className="card-img-top" src={props.url} alt={props.alternate} />
           <button>{props.meal}</button>
         </div>
         <div className="card-body">
           <div className="card-title">{props.title}</div>
-          <p className="card-text">{props.text}</p>
+          <p className="card-snippet">{props.snippet}</p>
+        </div>
+        <div className="card-bottom">
+          <span>{mealDifficulty()}{props.difficulty}</span>
```

Comment:
Might be a little more clear to pass the difficulty as an argument, like `mealDifficulty(props.difficulty)`, and adjust the function above to take an argument instead of using `props`.

Reply:
This is taken care of, as well as the variable name spacing with difficulty naming issue!